docs/start/getting_started.md

```diff
@@ -0,0 +1,52 @@
+# Getting started
+This guide will help you with the installation of flowchem.
+
+## Overview
+In a typical scenario, flowchem:
+1. reads a configuration file listing the devices to be controlled and their settings;
```

Comment:
this part is hard to get... short introduction for flowchem is good enough. Directly starting talking config file is confusing